pubsublite/pscompat/subscriber.go

```diff
@@ -58,6 +59,11 @@ func (ah *pslAckHandler) OnNack() {
 		return
 	}
 
+	// Ignore nacks for partitions that have been assigned away.
```

Comment:
Is this behavior replicated in the other clients? I think it's reasonable, but we should try to be consistent.

Reply:
Yes, both Java and Python do this. It was easier on those client libraries, due to their structure, but required exposing `PartitionActive()` from the wire layer here.